nodebuilder/tests/blob_test.go

```diff
@@ -19,9 +19,9 @@ import (
 )
 
 func TestBlobModule(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 25*time.Second)
 	t.Cleanup(cancel)
-	sw := swamp.NewSwamp(t)
+	sw := swamp.NewSwamp(t, swamp.WithBlockTime(time.Second*1))
```

Comment:
again with the https://github.com/celestiaorg/celestia-app/issues/2199 issue, we had to increase times. The total timeout has been decreased since when the test works it only takes ~8 seconds or so on my machine. We can increase tho if desired.

Reply:
Does this entirely solve the flake?

Reply:
it does on my machine locally (have ran it 20 times) and hasn't failed yet in CI either so it appears to be ok